code/datums/objective.dm

```diff
@@ -1184,24 +1192,87 @@ ABSTRACT_TYPE(/datum/objective/conspiracy)
 /datum/objective/conspiracy/dresscode
 	explanation_text = "Write up a new dress code for the station and enforce it on all crew."
 
-/datum/objective/conspiracy/dnd
-	explanation_text = "Start a D&D campaign and force crewmembers to participate."
+// /datum/objective/conspiracy/dnd
+// 	explanation_text = "Start a D&D campaign and force crewmembers to participate."
 
-/datum/objective/conspiracy/play
-	explanation_text = "Organize a play or musical and persuade crewmembers to participate by offering notions of fame and grandeur."
+// /datum/objective/conspiracy/play
+// 	explanation_text = "Organize a play or musical and persuade crewmembers to participate by offering notions of fame and grandeur."
 
 /datum/objective/conspiracy/flat
 	explanation_text = "Convince the crew that the station and in fact all of space is flat."
 
-/datum/objective/conspiracy/heisenbee
-	explanation_text = "Explain to the crew how, yes, Heisenbee really was framed by the Chompski brothers."
+// /datum/objective/conspiracy/heisenbee
+// 	explanation_text = "Explain to the crew how, yes, Heisenbee really was framed by the Chompski brothers."
 
 /datum/objective/conspiracy/centcom
 	explanation_text = "Convince the crew that Central Command has forsaken them."
 
 /datum/objective/conspiracy/spacelaw
 	explanation_text = "Establish and enforce a set of station protocols and policies."
 
+/datum/objective/conspiracy/discountdan
+	explanation_text = "Transfer ownership of the station to Discount Dan. Ensure all the crew are loyal and the station is branded correctly."
```

Comment:
one suggestion: this, but the syndicate. possibly other factions? get some wizards in here?

Reply:
There's already "Convince the crew to forsake Nanotrasen and join the Syndicate.", but I've added "the Space Wizards Federation" and "Bombini"